README.md

```diff
@@ -1,73 +1,122 @@
 
-- ✅ **Contract Hot Reload**: Your frontend auto-adapts to your smart contract as you edit it.
-- 🔥 **Burner Wallet & Local Faucet**: Quickly test your application with a burner wallet and local faucet.
-- 🔐 **Integration with Wallet Providers**: Connect to different wallet providers and interact with the Ethereum network.
+To learn more about base Scaffold-ETH 2 features and development guide, check our [Docs](https://docs.scaffoldeth.io/) and [Website](https://scaffoldeth.io/).
 
-![Debug Contracts tab](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/1171422a-0ce4-4203-bcd4-d2d1941d198b)
+## 🏃 Quick Start
 
-## Requirements
+To get started with Scaffold-ETH 2 PWA, follow the steps below:
 
-Before you begin, you need to install the following tools:
+### 1 . Clone this repo & install dependencies
 
-- [Node (v18 LTS)](https://nodejs.org/en/download/)
-- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
-- [Git](https://git-scm.com/downloads)
+```bash
+gh repo clone BuidlGuidl/PWA-burner-wallet
+cd PWA-burner-wallet
+yarn install
+```
 
-## Quickstart
+### 2 . Setting up Firebase
 
-To get started with Scaffold-ETH 2, follow the steps below:
+> Note : You can also use other database as well, we are using Firebase for this example checkout `packages/nextjs/database/firebase`.
 
-1. Clone this repo & install dependencies
+Copy `packages/nextjs/.env.example` into `packages/nextjs/.env` file and fill in your Firebase credentials which starts with `FB`.
 
-```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
+Checkout [this article](https://softauthor.com/add-firebase-to-javascript-web-app/#add-firebase-sdk-to-javascript-web-app) on how to get config variables from Firebase.
+
+> Hint: If it's your first time using Firebase, please read the article from the beginning, as you'll need to create a Firebase project and register a web app first.
+
+We use the Firebase config variables in our `packages/nextjs/database/firebase/config` file:
+
+```js
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: process.env.FB_API_KEY,
+  authDomain: process.env.FB_AUTH_DOMAIN,
+  projectId: process.env.FB_PROJECT_ID,
+  storageBucket: process.env.FB_STRG_BUCKET,
+  messagingSenderId: process.env.FB_MESSAGING_SENDER_ID,
+  appId: process.env.FB_APP_ID,
+};
 ```
 
-2. Run a local network in the first terminal:
+> Hint : Make sure you have read and write [rules](https://firebase.google.com/docs/firestore/security/get-started#testing_rules) allowed in Firebase.
 
+### 3 . Setting VAPID Keys
+
+Run the following command to generate Public and Private VAPID :
+
+```bash
+yarn web-push-generate
 ```
-yarn chain
```

Comment:
I think we need chain & deploy, right?

If not you get an error when launching the app locally on hardhat.

Reply:
Just added the info to setup the local environment to the _"Starting the PWA"_ section.  

To keep it reduced, used the guide we use on SRE challenges, assuming users can read more details in SE-2 Docs, but not 100% sure if that assumption/approach is correct.

**I think could be interesting to add an extra section with the info on how to deploy to a real network**, even a reduced explanation could do the job, plus linking to the Docs sections if the user wants extra info (Deploy your Smart Contracts and Deploy your NextJS App sections). 

What do you think?

Reply:
Yep, I agree!

I think this is ready to merge. I'll set up a Vercel with auto-deployment + firebase for this and will merge to main. Probably next week, so feel free to add that section if you want @Pabl0cks (if not, it's not a big deal and we can iterate on a future PR)

Thanks!

Reply:
Section added! 🙌 
Was not sure about adding or not an initial sentence on why would you want to deploy to a public network, but I guess anyone building a PWA with SE-2 already knows why.